README.md

```diff
@@ -1,2 +1,148 @@
 # Hello-Microverse
-In this project, I'm gonna test my understanding of the previous lessons.
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+> In this project, I'm gonna test my understanding of the previous lessons.
+
+## 🛠 Built With <a name="built-with"></a>
+> Html & CSS
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> features of the application.
+
+- **[key_feature_1]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://ouail-01.github.io/Hello-Microverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+### Usage
+
+### Run tests
+
+
+### Deployment
```

Comment:
- [ ] You can provide information on how users can install and use the project in their local machines but remove the `run tests` header since you are not running tests in this particular milestone. Try as much as possible to customize your readme file to contain the information that is available